app/emails/templates/reset_password_email_locked_account_orams.md

```diff
@@ -0,0 +1,9 @@
+# Reset your ORAMS password
```

Comment:
isn't the template identical except for the word `ORAMS`?

Reply:
For ORAMS, I also removed `contact-us` because we weren't sure how support was going to be handled